src/main/target/OMNIBUSF7/target.h

```diff
@@ -155,13 +169,19 @@
 
 #define SDCARD_DMA_STREAM_TX_FULL           DMA2_Stream1
 #define SDCARD_DMA_CHANNEL                  4
+//FLASH----------------------------------------
```

Comment:
Build and initialisation will be a bit faster if we only define SD card / onboard flash for the individual variants.